crates/std_detect/src/detect/mod.rs

```diff
@@ -135,3 +135,10 @@ pub fn features() -> impl Iterator<Item = (&'static str, bool)> {
     }
     impl_()
 }
+
+#[test]
```

Comment:
Please put tests in the tests/ directory -this gets included in libstd as is.

Reply:
Moved everything now.